src/backend/aspen/app/views/tests/test_samples_view.py

```diff
@@ -25,7 +25,13 @@ def test_samples_view(
     group = group_factory()
     user = user_factory(group)
     sample = sample_factory(group)
-    uploaded_pathogen_genome = uploaded_pathogen_genome_factory(sample)
+    pathogen_genome = uploaded_pathogen_genome_factory(
+        sample,
+        pangolin_lineage="B.1.590",
+        pangolin_probability=1.0,
+        pangolin_version="2021-04-23",
+        pangolin_last_updated=datetime.datetime.now(),
```

Comment:
Consider adding this to the factory so it doesn't have to be repeated so many times.

Reply:
added

Reply:
done